lib/record/bluetooth.dart

```diff
@@ -767,19 +769,20 @@ class BluetoothEasyPairingRecord extends BluetoothRecord {
   @override
   String toString() {
     var str = "BluetoothEasyPairingRecord: ";
-    str += "address=${address.address} ";
+    // str += "address=${address?.address} ";
+    str += "address=${(address?.address) == null ? " address is null " : address?.address} ";
```

Comment:
Replace with cleaner code:

```suggestion
    str += "address=${address?.addresss ?: '(null)'} ";
```